LCM/dsc/engine/eventing/oidsc.h

```diff
@@ -892,3 +892,24 @@ FILE_EVENT2(4331, MessageStartingDscOperation_Impl, LOG_WARNING, PAL_T("Job %s :
 #define MessageDscOperationCompleted(a0, a1, a2) MessageDscOperationCompleted_Impl(0, 0, tcs(a0), tcs(a1), tcs(a2))
 #endif
 FILE_EVENT3(4332, MessageDscOperationCompleted_Impl, LOG_WARNING, PAL_T("Job %s : %s DSC operation completed in %s seconds."), const TChar*, const TChar*, const TChar*)
+
+#if defined(CONFIG_ENABLE_DEBUG)
+#define MessageDscEngineProviderLoaded(a0) MessageDscEngineProviderLoaded_Impl(__FILE__, __LINE__, tcs(a0))
+#else
+#define MessageDscEngineProviderLoaded(a0) MessageDscEngineProviderLoaded_Impl(0, 0, tcs(a0))
+#endif
+FILE_EVENT1(4333, MessageDscEngineProviderLoaded_Impl,LOG_WARNING, PAL_T("Job %s : \nLoading DSC Engine Provider"), const TChar*)
+#if defined(CONFIG_ENABLE_DEBUG)
+#define MessageDscEngineProviderUnLoaded(a0) MessageDscEngineProviderUnLoaded_Impl(__FILE__, __LINE__, tcs(a0))
+#else
+#define MessageDscEngineProviderUnLoaded(a0) MessageDscEngineProviderUnLoaded_Impl(0, 0, tcs(a0))
+#endif
+FILE_EVENT1(4334, MessageDscEngineProviderUnLoaded_Impl, LOG_WARNING, PAL_T("Job %s : \nUnLoading DSC Engine Provider"), const TChar*)
+#if defined(CONFIG_ENABLE_DEBUG)
+#define MessageDscEngineRegisteringSignalHandler(a0) MessageDscEngineRegisteringSignalHandler_Impl(__FILE__, __LINE__, tcs(a0))
+#else
+#define MessageDscEngineRegisteringSignalHandler(a0) MessageDscEngineRegisteringSignalHandler_Impl(0, 0, tcs(a0))
+#endif
+FILE_EVENT1(4335, MessageDscEngineRegisteringSignalHandler_Impl, LOG_WARNING, PAL_T("Job %s : \nRegistered Process Signal Handler for SIGCHLD Event"), const TChar*)
```

Comment:
>LOG_WARNING [](start = 65, length = 11)

This shouldn't be warning and same for above as well.

Reply:
we seem to be logging all messages as LOG_WARNING. if I make it LOG_INFO will it go to omsconfig logs then right we dont want that right ?

Reply:
e.g why is this message MessageDscOperationCompleted_Impl, LOG_WARNING

Reply:
I think it was either done by mistake (a copy paste ) or the intention was to always populate this message in omsconfig.log, not sure. For loading/unloading DSC Engine provider falls in the same category and having them in omsconfig seems appropriate.

---
In reply to: [177270516](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/380#discussion_r177270516) [](ancestors = 177270516)

Reply:
Last 2 events may be too much data for omsconfig.

---
In reply to: [177271250](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/380#discussion_r177271250) [](ancestors = 177271250,177270516)